Source/Dafny/Dafny.atg

```diff
@@ -2055,25 +2064,30 @@ FunctionSpec<.List<AttributedExpression/*!*/>/*!*/ reqs, List<FrameExpression/*!
      Contract.Requires(cce.NonNullElements(reads));
      Contract.Requires(decreases == null || cce.NonNullElements(decreases));
      Expression/*!*/ e;  FrameExpression/*!*/ fe;
-     Attributes ensAttrs = null; Attributes reqAttrs = null; .)
+     Attributes ensAttrs = null; Attributes reqAttrs = null;
```

Comment:
It seems that `ensAttrs`, `reqAttrs`, `readsAttrs` (L2073), and `decrAttrs` (L2086) can all use the same variable (whose name could be just `attrs`).

Reply:
Yes - changed it here and elsewhere